chemprop/__init__.py

```diff
@@ -1,3 +1,3 @@
 from . import data, featurizers, models, nn, utils, conf, exceptions, schedulers
 
-__version__ = '2.0.0b1'
+__version__ = "2.0.0b1"
```

Comment:
@kevingreenman I believe we use bump version to automatically change this string. Will there be a problem if we use " instead of '?

Reply:
I don't think that will cause any problems, thanks for checking.